plugins/babel/test/tasks/babel.test.ts

```diff
@@ -30,6 +24,6 @@ describe('babel', () => {
     )
     await task.run()
     const transpiled = await fs.readFile(path.join(outputPath, 'index.js'), 'utf8')
-    expect(transpiled).toEqual(transpiledFixture)
+    expect(transpiled).toMatchSnapshot()
```

Comment:
**suggestion**: this snapshot is small enough i think it's probably cleaner to use an inline snapshot?

Reply:
tbh the fixture is small enough it could probably be just a string